src/ORM/Marshaller.php

```diff
@@ -198,17 +199,18 @@ public function one(array $data, array $options = [])
             }
         }
 
-        if (!isset($options['fieldList'])) {
-            $entity->set($properties);
-            $entity->errors($errors);
-
-            return $entity;
+        if (isset($options['fieldList'])) {
+            $options['fields'] = $options['fieldList'];
```

Comment:
I am not sure the deprecated option should overwrite the undeprecated one. Perhaps this should be `isset($options['fieldList']) && !isset($options['fields'])`?